steam_app/controllers/FavoritesViewController.swift

```diff
@@ -8,26 +8,54 @@
 import UIKit
 
 class FavoritesViewController: UIViewController {
-    let favsList = LocalCache.favsList
     @IBOutlet weak var tableView: UITableView!
-
+    @IBOutlet weak var search: UISearchBar!
+    let favsList = LocalCache.favsList
+    var filteredData = [Favorite]()
```

Comment:
it's a bad approach to put an instance of the Game inside the array as the array will never be empty.
Better to put it this way var filteredData: [Favorite] = []